packages/venia-concept/src/components/SearchBar/__tests__/searchBar.spec.js

```diff
@@ -119,8 +120,11 @@ test('the reset button clears the input', () => {
     const button = instance.find(buttonTypes);
     button.props.onClick();
 
-    // Test that the input has been cleared.
+    // Test that the input has been cleared and autocomplete is closed.
     expect(input.props.value).toBe('');
+    expect(
+        instance.findByType(SearchAutocomplete).props.autocompleteVisible
```

Comment:
This test should look at the state right? Or you can mock the `updateAutocompleteVisible` function and assert that it was called with `false`.

Reply:
Scratch that. I see you are testing the child `SearchAutocomplete` component's props.